force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -159,10 +208,10 @@ public with sharing class VariablesDataTypesOperators {
      * @param minutes The number of minutes.
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
-    Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
+    final static Integer MILLISECONDS_PER_MINUTE = 60000; // Make this value a constant
```

Comment:
Try to see if you can move this variable to the top of the class, outside this method